Hazel/src/Hazel/Renderer/OrthographicCameraController.h

```diff
@@ -21,6 +21,11 @@ namespace Hazel {
 
 		float GetZoomLevel() const { return m_ZoomLevel; }
 		void SetZoomLevel(float level) { m_ZoomLevel = level; }
+		
+		const void increasePosition(float x, float y) { m_CameraPosition.x += x; m_CameraPosition.y += y; }
+		const void decreasePosition(float x, float y) { m_CameraPosition.x -= x; m_CameraPosition.y -= y; }
+		const void increaseRotation(float rotation) { m_Rotation += rotation; }
+		const void decreaseRotation(float rotation) { m_Rotation -= rotation; }
```

Comment:
The names of the functions don't follow the Hazel convention of starting with the uppercase letter. These functions also don't actually update the camera's position. They only modify the controller's variables.

I am also not sure if these functions are really necessary. If we do want to include these we should maybe also make it clear whether this change is in camera space or in world space (like whether calling `IncreasePosition(0.0f, 1.0f)` would move in the camera's up direction or just add 1 to the y coordinate).

I also think we should rather have only one `ChangePosition` method (if we are adding this), since floats can be negative anyway and having both an increase and decrease method isn't needed. Even if we want these methods, should it be part of this PR?

Reply:
Yea, ``ChangePosition'' its better name. I originally wanted to call it that but then I thought that ``Increase, Decrease`` would be clearer.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/178/commits/fb258084c6f1397579435b42d8daf5377d4dcfb7.